tests/integration/SdkTest_test.cpp

```diff
@@ -833,7 +872,7 @@ void SdkTest::releaseMegaApi(unsigned int apiIndex)
     assert(megaApi[apiIndex].get() == mApi[apiIndex].megaApi);
     if (mApi[apiIndex].megaApi)
     {
-        if (mApi[apiIndex].megaApi->isLoggedIn())
+        if (mApi[apiIndex].megaApi->isLoggedIn() && !gResumeSessions)
         {
             ASSERT_NO_FATAL_FAILURE( logout(apiIndex) );
         }
```

Comment:
I figured that the full login/logout is the normal way to run it, in jenkins  - I just wanted to speed it up when running the tests manually - it saves around 20 or 30 seconds per test when running the whole suite.    If we locallogout when the next test will do a full login(), we'll be building up sessions that may never be closed.    I think we are calling locallogout() when the next test will resume the session.   thanks

Reply:
I think that's not correct. A locallogout does not close the session and, by default, it keeps the local cache available to resume the session later, by providing the session id. The locallogout perform, however, a regular cleanup of the client, closing everything that might be in flight, etc.
Whether you call locallogout or not, the risk to leave orphan sessions is there, simply because if !gResumeSessions, the session won't be closed.

Reply:
Ah, I misread this initially sorry, yes no problem calling locallogout() here in that case - I'll commit the suggestion